test/events-specs.js

```diff
@@ -42,6 +44,7 @@ describe('events', () => {
           response.body[0].team_limit.should.equal(e1.team_limit)
           response.body[0].team_building_start.should.be.sameMoment(e1.team_building_start)
           response.body[0].team_building_end.should.be.sameMoment(e1.team_building_end)
+          response.body[0].default_steps.should.be.equal(e1.default_steps)
```

Comment:
Looks like this is supposed to be `should.equal`
```suggestion
          response.body[0].default_steps.should.equal(e1.default_steps)
```